package.json

```diff
@@ -151,7 +150,8 @@
   },
   "babel": {
     "plugins": [
-      "lodash", [
+      "lodash",
+      [
```

Comment:
lol. maybe -> lint changes it, npm rewrites it.. what's up with the bracket placement..